.gitignore

```diff
@@ -1,3 +1,2 @@
 .idea
-composer.lock
```

Comment:
That advice is outdated, please see this post by Nils Adermann: https://blog.engineyard.com/2015/composer-continuous-integration